layouts/DocsPage/Versions.tsx

```diff
@@ -2,30 +2,90 @@ import cn from "classnames";
 import { useRouter } from "next/router";
 import { useCallback, useEffect, useMemo, useState } from "react";
 import { Dropdown } from "components/Dropdown";
-import type { VersionsInfo } from "./types";
+import type { VersionsInfo, VersionsDropdown } from "./types";
 import styles from "./Versions.module.css";
 
-const renderVersion = (version: string) => `Version ${version}`;
+// renders strikethrough for deprecated versions
+const RenderVersion = (version: VersionsDropdown) => {
```

Comment:
Can you make this lowercase 'r' since it's not technically a component? (it doesn't take in props, though it does return `jsx`).

Reply:
updated